asmcomp/selectgen.ml

```diff
@@ -288,6 +292,9 @@ method select_operation op args =
   | (Cfloatofint, _) -> (Ifloatofint, args)
   | (Cintoffloat, _) -> (Iintoffloat, args)
   | (Ccheckbound _, _) -> self#select_arith Icheckbound args
+  | (Cperform, _) -> (Iperform, args)
+  | (Cresume, _) -> (Iresume_ind, args)
```

Comment:
The distinction between `Iresume_ind` and `Itail_resume_ind` is never made. Should this be similar to `Icall_ind` at tail position [0]?

[0] https://github.com/ocamllabs/ocaml-multicore/blob/effects-refactor/asmcomp/selectgen.ml#L728-L744